lib/rubocop/cop/style/quoted_symbols.rb

```diff
@@ -93,7 +93,7 @@ def correct_quotes(str)
                        end
 
           # The conversion process doubles escaped slashes, so they have to be reverted
-          correction.gsub('\\\\', '\\')
+          correction.gsub('\\\\', '\\').gsub('\"', '"')
```

Comment:
Can you add a test for the change?

Reply:
That change was made to keep [this](https://github.com/rubocop/rubocop/blob/master/spec/rubocop/cop/style/quoted_symbols_spec.rb#L116) test passing. It was the only one to fail after the change was made in `lib/rubocop/cop/util.rb`